perf/README

```diff
@@ -60,3 +60,13 @@ threadcount - The number of threads to perform handshakes on in the test
 
 The output is two values: the average time taken for a handshake in us, and the
 average handshakes per second performed over the course of the test.
+
+x509storeissuer
+---------------
+
+Runs the function call X509_STORE_CTX_get1_issuer() repeatedly in a loop (which
+is used in certificate chain building as part of a verify operation). The test
+assumes that the default certificates directly exists but is empty. For a
+default configuration this is "/usr/local/ssl/certs". The test takes the number
+of threads to use as an argument and the test reports the average time take to
+execute a block of 100 X509_STORE_CTX_get1_issuer() calls.
```

Comment:
Strictly, this isn't what it's doing.  It running everything then dividing the total time by number of blocks.  It should be running each block separately and averaging the times for each block.

Arithmetic mean probably isn't a good approach for a rate.  See e.g. [Notes on Calculating Computer Performance](https://tnm.engin.umich.edu/wp-content/uploads/sites/353/2021/06/1995_Notes_on_calculating_computer_performance.pdf).

Reply:
> Strictly, this isn't what it's doing. It running everything then dividing the total time by number of blocks. It should be running each block separately and averaging the times for each block.

I've changed it to do that. Possibly we might want to circle back over some of the other tests I've already put in and make a similar change.

> Arithmetic mean probably isn't a good approach for a rate.

I'd like to defer that to some future change.

Reply:
Sounds good.